src/stuf/templates/stuf/soap_envelope.xml

```diff
@@ -0,0 +1,17 @@
+{% load stuf %}<?xml version="1.0" encoding="UTF-8"?>
+{# TODO: use only relevant namespaces #}
+<soapenv:Envelope
+    xmlns:soapenv="{% if soap_version == '1.1' %}http://schemas.xmlsoap.org/soap/envelope/{% else %}http://www.w3.org/2003/05/soap-envelope{% endif %}"
+    xmlns:StUF="http://www.egem.nl/StUF/StUF0301"
+    xmlns:xlink="http://www.w3.org/1999/xlink"
+    xmlns:gml="http://www.opengis.net/gml"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:ns="{{ main_namespace }}"
```

Comment:
dirty solution -> need to find a nicer way, because the prefixes are actually used in the rest of the template code.

Reply:
Resolved via 0126688d73541bbe5ff157b1114d35cc71426dec